Dockerfile_multi_folder

```diff
@@ -0,0 +1,14 @@
+ARG BASE_IMAGE=nansencenter/geospaas:2.0.0-slim
```

Comment:
Comment on the whole file: a few empty lines to separate the sections would be nice.

Reply:
Done in the next commit.